pkg/util/kvgraph/backup_graph_test.go

```diff
@@ -240,7 +252,8 @@ func TestNewVirtualMachineBackupGraph(t *testing.T) {
 		vm       kvcore.VirtualMachine
 		expected []velero.ResourceIdentifier
 	}{
-		{"Should include all related resources",
+		{
```

Comment:
urgh sorry nvim decided to reformat all of this, I'll back this out once #334 lands and I rebase this series.